package-lock.json

```diff
@@ -1,7606 +0,0 @@
-{
```

Comment:
Decided to go with yarn over npm. Don't have any strong personal preferences for either so if the general consensus is to use npm then let's do that.

Reply:
I do not have a preference. We can revisit. I just might need a short refresher in the README on how to get the application compiled and started with yarn.

Reply:
Cool, will make necessary changes to the README.

Reply:
Have altered README to mention usage of yarn instead of npm. That should be enough for now.